package.json

```diff
@@ -5,7 +5,8 @@
 	"license": "0BSD",
 	"engines": {
 		"node": ">=10.18",
-		"yarn": ">=1.0.0"
+		"yarn": ">=1.0.0",
+		"npm": "please-use-yarn"
```

Comment:
"npm": "please-use-yarn".
npm>=7 ?

Reply:
Yeah, but the repository is using `yarn.lock` and `npm` will ignore `yarn.lock` and create `package-lock.json`; hence this will remove the advantage of having lock file

Reply:
The idea is clear, but I've never seen this kind of restriction anywhere else) Maybe it would be enough just to add `package-lock.json` to git ignore instead?

Reply:
@antongolub I revert this change